src/PostgreSQLCopyHelper.NodaTime.Test/PostgreSQLCopyHelper.NodaTime.Test.csproj

```
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
```

netcore 3.0 is technically EOL, we should probably make this 3.1
```suggestion
    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```